dist/api/index.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var fs = require('fs');
+var _f = require('fs-utils');
+var path = require('path');
+var cli = process.AURELIA;
+var API = function API() {};
+
+fs.readdirSync(__dirname).forEach(function (file) {
```

Comment:
The way we are building the API object. There are two aspect of this:
- Every time a user run a command this script will search the folder and build the API
- Any user/human consuming the API would not get a clear picture of what he is consuming
- API is a contract between `Consumer` and `Producer` This should be as explicit as possible. I should not change dynamically.